src/ontology/cl-edit.owl

```diff
@@ -4409,6 +4409,7 @@ SubClassOf(obo:CL_0000109 obo:CL_0000540)
 
 # Class: obo:CL_0000110 (peptidergic neuron)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/B978-012589762-4/50021-9") obo:IAO_0000115 obo:CL_0000110 "A neuron that uses short peptides as transmitters.")
```

Comment:
The dbxref is behind a paywall. Just looking at the abstract, there is no reference of short peptides (as neuropeptides are not defined). This characteristic is also not present in the Fbbt term. This reference might be useful to complement [PMID:34924955](https://pubmed.ncbi.nlm.nih.gov/34924955).

Reply:
Let’s use “neuropeptide” rather than “short peptides”, along with that reference instead: [PMCID:PMC3918222](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3918222/) (it’s mammalian-centric, but for lack of a true taxon-neutral reference a mammalian-centric one is better than a _C. elegans_ one, since Uberon/CL already has a strong mammalian bias anyway.

As for complementing the definition (using either reference), I disagree. The term is intended to be very generic, similar to all the others -ergic neurons in CL.